src/main/java/com/dev/cinema/model/dto/MovieSessionDtoRequest.java

```diff
@@ -1,7 +1,7 @@
 package com.dev.cinema.model.dto;
 
-import jakarta.validation.constraints.NotNull;
-import jakarta.validation.constraints.PositiveOrZero;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.PositiveOrZero;
 
 public class MovieSessionDtoRequest {
     @PositiveOrZero
```

Comment:
it is not a good idea to allow `0` id for movie or cinemaHall

Reply:
Ok. Thanx